.github/workflows/publish-pypi.yaml

```diff
@@ -35,3 +35,4 @@ jobs:
         uses: JRubics/poetry-publish@v1.16
         with:
           pypi_token: ${{ secrets.PYPI_API_TOKEN }}
+          plugins: "poetry-dynamic-versioning"
```

Comment:
need to install poetry-dynamic-versioning when building package
https://github.com/JRubics/poetry-publish/blob/dffe3aca4d4fc24f85410ac28815736c518223a4/action.yml#L43C3-L45

Reply:
@sai-g thank you. I have checked this on local using act and seems working as expected.
```
% git tag v0.0.3
% act --container-architecture linux/amd64 -j build-and-publish --pull=false
WARN[0000] Could not find any stages to run. View the valid jobs with `act --list`. Use `act --help` to find how to filter by Job ID/Workflow/Event Name
WARN[0000] Could not find any stages to run. View the valid jobs with `act --list`. Use `act --help` to find how to filter by Job ID/Workflow/Event Name
[Publish python package/build-and-publish] 🚀  Start image=catthehacker/ubuntu:act-latest
INFO[0000] Parallel tasks (0) below minimum, setting to 1
[Publish python package/build-and-publish]   🐳  docker pull image=catthehacker/ubuntu:act-latest platform=linux/amd64 username= forcePull=false
INFO[0002] Parallel tasks (0) below minimum, setting to 1
[Publish python package/build-and-publish]   🐳  docker create image=catthehacker/ubuntu:act-latest platform=linux/amd64 entrypoint=["tail" "-f" "/dev/null"] cmd=[] network="host"
[Publish python package/build-and-publish]   🐳  docker run image=catthehacker/ubuntu:act-latest platform=linux/amd64 entrypoint=["tail" "-f" "/dev/null"] cmd=[] network="host"
[Publish python package/build-and-publish]   ☁  git clone 'https://github.com/JRubics/poetry-publish' # ref=v1.16
[Publish python package/build-and-publish] ⭐ Run Main actions/checkout@v4
[Publish python package/build-and-publish]   🐳  docker cp src=/ws/hiohiohio-alpaca-py/. dst=/ws/hiohiohio-alpaca-py
[Publish python package/build-and-publish]   ✅  Success - Main actions/checkout@v4
[Publish python package/build-and-publish] ⭐ Run Main Build and publish to pypi
[Publish python package/build-and-publish]   🐳  docker pull image=jrubics/poetry-publish:v1.16 platform=linux/amd64 username= forcePull=false
[Publish python package/build-and-publish]   🐳  docker create image=jrubics/poetry-publish:v1.16 platform=linux/amd64 entrypoint=[] cmd=["latest" "latest" "" "" "" "" "" "" "" "" "" "poetry-dynamic-versioning" "."] network="container:act-Publish-python-package-build-and-publish-6f9d5ebb146892ab87d15a24d59ba55efc98024e43d94191a329bcad6363d930"
[Publish python package/build-and-publish]   🐳  docker run image=jrubics/poetry-publish:v1.16 platform=linux/amd64 entrypoint=[] cmd=["latest" "latest" "" "" "" "" "" "" "" "" "" "poetry-dynamic-versioning" "."] network="container:act-Publish-python-package-build-and-publish-6f9d5ebb146892ab87d15a24d59ba55efc98024e43d94191a329bcad6363d930"
...
| Using version ^1.2.0 for poetry-dynamic-versioning
...
| Package operations: 4 installs, 0 updates, 0 removals
|
|   • Installing markupsafe (2.1.3)
|   • Installing dunamai (1.19.0)
|   • Installing jinja2 (3.1.2)
|   • Installing poetry-dynamic-versioning (1.2.0)
...
Installing the current project: alpaca-py (0.0.3)
| Building alpaca-py (0.0.3)
|   - Building sdist
|   - Built alpaca_py-0.0.3.tar.gz
|   - Building wheel
|   - Built alpaca_py-0.0.3-py3-none-any.whl
|
| Expected a value for pypi-token.pypi setting.
[Publish python package/build-and-publish]   ❌  Failure - Main Build and publish to pypi
[Publish python package/build-and-publish] exit with `FAILURE`: 1
[Publish python package/build-and-publish] 🏁  Job failed
Error: Job 'build-and-publish' failed
```